### ISSUE TITLE
Fields with a `default: ..` value keep getting that value. 

### ISSUE THREAD

**Issue by bobdenotter** [bolt/bolt#5795]
A field with a `default: ..` value should get that value when creating a new record. Currently they also get this value when editing an existing record in the backend.

Details
-------

 - Relevant Bolt Version: [ 3.0 and up ]

Reproduction
------------

Create a simple contenttype, like this: 

```
dummy:
    name: Dummies
    singular_name: Dummy
    fields:
        title:
            type: text
            class: large
        slug:
            type: slug
            uses: [ title ]
        content:
            type: textarea
            default: Pompidom
```

Create a new record, and you'll this, as expected: 

![screen shot 2016-09-21 at 13 06 48](https://cloud.githubusercontent.com/assets/1833361/18708674/5bab324e-7ffc-11e6-9e64-e8742291cb43.png)

Edit the fields, but _empty_ the textarea fields and save: 

![screen shot 2016-09-21 at 13 07 18](https://cloud.githubusercontent.com/assets/1833361/18708682/6e01090a-7ffc-11e6-91c2-f8b5bf3d8091.png)

The database will contain the expected results: 

![screen shot 2016-09-21 at 13 09 10](https://cloud.githubusercontent.com/assets/1833361/18708739/a2830afc-7ffc-11e6-9a56-fb8784cd3dce.png)

Refresh the page, and the `content` will again contain the `default` value. 

![screen shot 2016-09-21 at 13 09 57](https://cloud.githubusercontent.com/assets/1833361/18708756/b6eeefa6-7ffc-11e6-885e-b12bd8380ff1.png)

**Comment by bobdenotter** [comment]
It seems to get the 'default' values at this point: https://github.com/bolt/bolt/blob/release/3.1/src/Storage/Repository.php#L433

I haven't figured out yet what happens exactly.. @rossriley, any ideas?

**Comment by bobdenotter** [comment]
Fixed by #5805
